<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="6.55s"
		durationms="6550"
		type="root"
		rid="443cf1acc6c6728"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="dgomeza"
		rxversion="9.3.3.0"
		host="MEDLAPDESPP2BTR"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="es-ES"
		dtnlanguage="en-US"
		screenresolution="1366x768"
		timestamp="12/09/2020 2:37:31 PM"
		timestampiso="2020-12-09T19:37:31"
		endtime="12/09/2020 2:37:37 PM"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:06.540"
			timeRelativeToTestModuleStartTime="00:06.540"
			timeWallClock="14:37:37"
			time="00:06.540"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
